tests/unit/fix_coverage_test.py

```diff
@@ -22,16 +22,16 @@ def test_fix_coverage(tmpdir):
         str(unrelated_file): {(5, 6): None},
     })
 
-    assert coverage_data.lines(base_file) == [1]
-    assert coverage_data.lines(sub_file) == [3]
-    assert coverage_data.lines(unrelated_file) == [5]
+    assert coverage_data.lines(base_file) == [1, 2]
+    assert coverage_data.lines(sub_file) == [3, 4]
+    assert coverage_data.lines(unrelated_file) == [5, 6]
 
     merge_coverage(coverage_data, '/site-packages/', str(tmpdir))
 
     # The base file should contain all the lines and arcs
-    assert coverage_data.lines(base_file) == [1, 3]
+    assert coverage_data.lines(base_file) == [1, 2, 3, 4]
     assert coverage_data.arcs(base_file) == [(1, 2), (3, 4)]
-    assert coverage_data.lines(unrelated_file) == [5]
+    assert coverage_data.lines(unrelated_file) == [5, 6]
     assert coverage_data.arcs(unrelated_file) == [(5, 6)]
```

Comment:
the implementation in coverage changed, I think with this commit:
https://bitbucket.org/ned/coveragepy/commits/b03f78976a782b70b6ac8fe1a2ac5b7263e9e355

anyway, this looks like it's still testing the merging functionality we want to